tests/end-to-end/vpc/App.cs

```diff
@@ -0,0 +1,52 @@
+using Amazon.CDK;
+using Amazon.CDK.AWS.EC2;
+using Constructs;
+using System.Collections.Generic;
+
+namespace Com.Acme.Test.Simple
+{
+    public class VpcStackProps : StackProps
+    {
+    }
```

Comment:
Should we omit these if no props are present?

Reply:
I have the same question. I am on the fence on this one. I decided to leave it in for two reasons:
1. Typescript and Go both do it this way
2. This shows the customer how they can start expanding their CDK app code to be more useful/flexible in a CDK idiomatic way

But, I can also see that it's adding clutter to the generated file.

Reply:
Fair. I think Hogan and I omitted it in python and java, but we can discuss as a follow up which version we should go with to ensure consistency. In Java it adds a lot of overhead so maybe we don't actually need to be consistent and just go with what's best to each language, but that still means it can be a fast follow up.

Reply:
I think omitting it is ideal, but if we aren't consistent I think that's totally fine. Especially since I don't think we will get much traction on our non TS/Python versions

Reply:
Okay, if we plan to omit from all the languages, then I've omitted it in c#. We can update the others in future changes.